packages/datadog_grpc_interceptor/test/src/generated/helloworld.pb.dart

```diff
@@ -0,0 +1,105 @@
+///
```

Comment:
if this is generated, should we commit it? same for files below

Reply:
In this case yes, I think so, since this isn't generated through the Dart build system and requires a both a separate tool (protoc) and a global pub package to generate it.  Also I should add that to the README.

For what it's worth, most other repos I've seen commit their generated gRPC classes.